application/models/Question.php

```diff
@@ -1132,4 +1132,17 @@ public function insertRecords($data)
         Yii::log(\CVarDumper::dumpAsString($oRecord->getErrors()), 'warning', 'application.models.Question.insertRecords');
     }
 
+    public function getQuestionAttribute($name, $default = null)
```

Comment:
Good function but

1. Need i18n for future
2. default (if not set here) must come from attribute definition

Reply:
Please use getQuestionAttributes() from model QuestionAttribute instead.